WordPress/Classes/Utility/Analytics/WPAppAnalytics.m

```diff
@@ -0,0 +1,202 @@
+//
+//  WPAppAnalytics.m
```

Comment:
Hey @diegoreymendez  - make sure to remove all header info from files checked into the project.